kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -180,4 +181,18 @@ private Object convertLogical(Object kafkaConnectObject,
         LogicalConverterRegistry.getConverter(kafkaConnectSchema.name());
     return converter.convert(kafkaConnectObject);
   }
+
+  private Double convertDouble(Object kafkaConnectObject) {
```

Comment:
I'm pretty sure there are unit tests for this class. We should definitely add some unit tests for this.

Reply:
Isn't there already a test for double values in BigQueryRecordConverterTest.testFloat()?